android/src/main/java/im/status/ethereum/keycard/SmartCard.java

```diff
@@ -310,6 +311,9 @@ public WritableMap getKeys(final String pairingBase64, final String pin) throws
         cmdSet.verifyPIN(pin).checkOK();
         Log.i(TAG, "pin verified");
 
+        byte[] tlvMaster = cmdSet.exportKey(MASTER_PATH, true, true).checkOK().getData();
```

Comment:
maybe we can set the second parameter to false because we don't need to set the master as current key.

Reply:
fixed